hackathon_site/registration/models.py

```diff
@@ -117,27 +135,59 @@ class Application(models.Model):
 
     q1 = models.TextField(
         null=False,
-        help_text="Why do you want to participate in NewHacks?",
-        max_length=1000,
+        help_text="How many hackathons have you been to?",
+        choices=HACKATHON_NUMBER_CHOICES,
     )
+
     q2 = models.TextField(
         null=False,
-        help_text="Tell us about your experience with software or other topics relevant to NewHacks!",
+        help_text="If you’ve been to a hackathon, briefly tell us your experience. If not, describe what you expect to see and experience. (1000 char max)",
         max_length=1000,
     )
+
     q3 = models.TextField(
-        null=False, help_text="How did you hear about NewHacks?", max_length=100
+        null=False,
+        help_text="Why do you want to participate in NewHacks? (1000 char max)",
+        max_length=1000,
+    )
+
+    q4 = models.TextField(
+        null=False,
+        help_text="What is your technical experience with software and hardware? (1000 char max)",
+        max_length=1000,
+    )
+
+    q5 = models.TextField(
+        null=False, help_text="How did you hear about NewHacks?", max_length=1000
     )
     conduct_agree = models.BooleanField(
         help_text="I have read and agree to the "
-        '<a href="https://docs.google.com/document/d/1Uec_PDknY-9nSMc7QOqSTFb54I71uGX6oZaoTm1u8Q0/">code of conduct</a>.',
+        '<a href="https://docs.google.com/document/d/1Uec_PDknY-9nSMc7QOqSTFb54I71uGX6oZaoTm1u8Q0/">MLH code of conduct</a>.',
```

Comment:
This is the MLH code of conduct: https://static.mlh.io/docs/mlh-code-of-conduct.pdf